configure.ac

```diff
@@ -1167,6 +1167,9 @@ AC_MSG_CHECKING([clang major version installed])
 if which llvm-config >/dev/null 2>&1
```

Comment:
this condition could be removed now

Reply:
+1 & yeah, new version of autoconf in f37 I guess.